webapp/src/main/webapp/src/app/pages/tests/create/test-create.page.ts

```diff
@@ -246,4 +256,8 @@ export class TestCreatePageComponent implements OnInit {
       }
     );
   }
+
+  public isRunnerAvailable(runner: string) : boolean{
```

Comment:
I saw you latest comment and seems all good. However, I think this function should be renamed to something like `isContractAvailable(contractType: string)` as we check the contract availability from a runner in the select option.

Reply:
Yup, made this change.